src/components/styles/calculator.css

```diff
@@ -1,9 +1,19 @@
+.calculate {
+  display: flex;
+
+  /* align-items: center; */
```

Comment:
- [ ] Please do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻